### ISSUE TITLE
Tutorial_2_General_Mixture_Models needs data rescaling

### ISSUE THREAD

**Issue by ngoix** [jmschrei/pomegranate#142]
Seems that a `data.reshape(-1, 1)` is needed before passing it to the mixture model.

 ```
ValueError                                Traceback (most recent call last)
<ipython-input-25-5d4a0e63559a> in <module>()
----> 1 labels = d.predict(data)
      2 labels

pomegranate/gmm.pyx in pomegranate.gmm.GeneralMixtureModel.predict (pomegranate/gmm.c:6452)()

pomegranate/gmm.pyx in pomegranate.gmm.GeneralMixtureModel.predict (pomegranate/gmm.c:6209)()

ValueError: need more than 1 value to unpack
```

**Comment by ngoix** [comment]
I installed pomegranate with pip btw.
Also I need to pass a numpy array of weights, not as described in the docs
http://pomegranate.readthedocs.io/en/latest/gmm.html
`gmm = GeneralMixtureModel([NormalDistribution(5, 2), NormalDistribution(1, 2)], weights=[0.33, 0.67])`

otherwise I get

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-6-3e7f9c152949> in <module>()
      7 
      8 #gmm = GeneralMixtureModel(NormalDistribution, n_components=3).fit(X)
----> 9 gmm = GeneralMixtureModel([NormalDistribution(5, 2), NormalDistribution(1, 2)], weights=[0.33, 0.67])
     10 
     11 preds = gmm.predict(X)

pomegranate/gmm.pyx in pomegranate.gmm.GeneralMixtureModel.__init__ (pomegranate/gmm.c:4599)()

AttributeError: 'list' object has no attribute 'sum'
```

**Comment by jmschrei** [comment]
The first is an issue, as we recently changed from allowing 1d arrays for 1d data to just making it 2d. Thanks for pointing that out. The second issue should be resolved by checking out the most recent version of the code.

**Comment by ngoix** [comment]
Ok thanks!

**Comment by jmschrei** [comment]
Should be resolved by #144
